sites/docs/src/pages/_ViewConfigEditor.js

```diff
@@ -71,14 +69,40 @@ export default function ViewConfigEditor(props) {
   const defaultViewConfigDocsUrl = useBaseUrl('/docs/default-config-json');
 
   const [pendingUrl, setPendingUrl] = useState('');
-  const [datasetUrls, setDatasetUrls] = useState('');
+  const [datasetUrls, setDatasetUrls] = useState('https://s3.amazonaws.com/vitessce-data/0.0.33/main/human-lymph-node-10x-visium/human_lymph_node_10x_visium.h5ad.zarr; https://vitessce-data.storage.googleapis.com/0.0.33/main/human-lymph-node-10x-visium/human_lymph_node_10x_visium.ome.zarr');
   const [pendingFileContents, setPendingFileContents] = useState('');
 
   const [syntaxType, setSyntaxType] = useState('JSON');
   const [loadFrom, setLoadFrom] = useState('editor');
 
   const exampleURL = 'https://assets.hubmapconsortium.org/a4be39d9c1606130450a011d2f1feeff/ometiff-pyramids/processedMicroscopy/VAN0012-RK-102-167-PAS_IMS_images/VAN0012-RK-102-167-PAS_IMS-registered.ome.tif';
 
+  const [debouncedHintsConfig, setDebouncedHintsConfig] = useState({});
+
+  function sanitiseURLs(urls) {
+    return urls
+      .split(/;/)
+      .map(url => url.trim())
+      .filter(url => url.match(/^http/g));
+  }
+
+  useEffect(() => {
+    const handle = setTimeout(() => {
+      const sanitisedUrls = sanitiseURLs(datasetUrls);
+      if (sanitisedUrls.length === 0) return;
+      try {
+        const newHintsConfig = getDatasetHintsConfig(sanitisedUrls);
+        setError(null);
+        setPendingJson(baseJson);
+        setDebouncedHintsConfig(newHintsConfig);
+      } catch (e) {
+        setError(e.message);
+      }
+    }, 500);
+
+    return () => clearTimeout(handle);
+  }, [datasetUrls, setPendingJson, setError]);
```

Comment:
Why is a timeout/debouncing needed? It seems to cause a delay in the UI

Reply:
I added the delay, because otherwise the UI will show an error that `One or more of the URLs provided point to unsupported file types.` while the user is still typing their URL.

Reply:
If the delay is not desirable, I can remove the debounce/timeout and instead add a button "get hints". What is your opinion?